Tasks/AzureMonitorAlerts/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "azuremonitoralerts",
+  "version": "0.0.1",
+  "description": "Azure monitor alerts",
+  "main": "azuremonitoralerts.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "dependencies": {
+    "q": "1.5.0",
+    "typed-rest-client": "^0.12.0",
```

Comment:
do not specify ^, that way, the version being installed will be updated without anyone's notice. This can be a problem when shrinkwrapping the task.

Reply:
Yeah, thanks ! Will update.